src/frameName.cpp

```diff
@@ -151,7 +155,10 @@ const char* FrameName::decodeNativeSymbol(const char* name) {
     }
 
     if (lib_name != NULL) {
-        snprintf(_buf, sizeof(_buf) - 1, "%s`%s", lib_name, name);
+        snprintf(_buf, sizeof(_buf) - 1, "%s (%s)", name, lib_name);
+        return _buf;
+    } else if (report_lib_name_from_symbol) {
+        snprintf(_buf, sizeof(_buf) - 1, "(%s)", name);
```

Comment:
What is `name` in this case, if `lib_name` is NULL?

Reply:
I mean, I figured it'll be the library name, but I don't understand the code in that case. Why is a variable named `lib_name` be NULL in that case? :sweat_smile:

Reply:
I guess it will be less confusing if I assign the library name back to `lib_name` and refer to it in this call.

Reply:
Yes. Better now